### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by valeryhime**
- The main Activity split into two tabs
- Added Fragments for Sender and Receiver tabs
- "Use playback capture" checkbox replaced with select
- Created copyable block for ports and IP addresses
- Plain text with instructions replaced with list of instructions
- Used colors from Roc icon
- Added playing status for sender and receiver tabs
- Added logs

### CODE REVIEW

**Comment by valeryhime** [comment]
Screenshot_20230122_144256
Screenshot_20230122_144303

**Comment by ortex** [comment]
Hi @valeryhime ! It looks great!

Could you please:
* rebase on main 
* run spotless `./gradlew :spotlessApply`

And I'll look closely in upcoming times 😄

**Comment by valeryhime** [comment]
Hi @ortex!

Done ^_^

File: app/src/main/java/org/rocstreaming/rocdroid/fragment/ReceiverFragment.kt [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            getText(R.string.receiver_sender_port_for_repair).toString().format(4)
+
+        view.findViewById<Button>(R.id.startReceiverButton).setOnClickListener {
+            startStopReceiver(view)
+        }
+
+        return view
+    }
+
+    fun startStopReceiver(@Suppress("UNUSED_PARAMETER") view: View) {
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/fragment/ReceiverFragment.kt:57]: let's make `startStopReceiver` private and remove unused param?

File: app/src/main/java/org/rocstreaming/rocdroid/fragment/SenderFragment.kt [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        Log.d(LOG_TAG, "Add Receiver State Changed Listener")
+
+        senderService = service
+
+        senderService.setSenderStateChangedListeners(
+            senderChanged = { state: Boolean ->
+                activity?.runOnUiThread {
+                    activity?.findViewById<Button>(R.id.startSenderButton)?.text =
+                        getText(if (state) R.string.stop_sender else R.string.start_sender)
+                    if (state) showActiveIcon(1) else hideActiveIcon(1)
```
[the remark this answers is not in the document]
Comment (ortex) [reply app/src/main/java/org/rocstreaming/rocdroid/fragment/SenderFragment.kt:123]: or, even better remove index from fragment at all:

calling:
```
            receiverFragment.onServiceConnected(
                senderReceiverService,
                { showActiveIcon(0) },
                { hideActiveIcon(0) }
            )
```

method signature:
```
    fun onServiceConnected(
        service: SenderReceiverService,
        onSenderStart: () -> Unit,
        onSenderStop: () -> Unit
    ) {
```

File: app/src/main/java/org/rocstreaming/rocdroid/SplashScreen.kt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import android.os.Bundle
+import android.os.Handler
+import androidx.appcompat.app.AppCompatActivity
+
+class SplashScreen : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_splash_screen)
+
+        var handler = Handler()
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/SplashScreen.kt:13]: nit:

var -> val 

😄

File: app/src/main/java/org/rocstreaming/rocdroid/adapter/ViewPagerAdapter.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.Lifecycle
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import org.rocstreaming.rocdroid.fragment.ReceiverFragment
+import org.rocstreaming.rocdroid.fragment.SenderFragment
+
+private const val NUM_TABS = 2
+
+public class ViewPagerAdapter(fragmentManager: FragmentManager, lifecycle: Lifecycle) :
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/adapter/ViewPagerAdapter.kt:12]: nit:
in kotlin classes are public by default, so `public` here is redundant and could be deleted

File: app/src/main/java/org/rocstreaming/rocdroid/component/CopyBlock.kt [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+        val view: View = inflate(context, R.layout.copy_block_component, this)
+
+        textBlock = view.findViewById<TextView>(R.id.block_label)
+
+        view.findViewById<ConstraintLayout>(R.id.copy_block).setOnClickListener {
+            setClipboard(context, findViewById<TextView>(R.id.block_label).text.toString())
+        }
+
+        view
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/component/CopyBlock.kt:34]: `view` could be deleted here, unused

**Comment by ortex** [comment]
And could you please update screenshots in readme? 😄

File: app/src/main/java/org/rocstreaming/rocdroid/fragment/SenderFragment.kt [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+        receiverIpEdit = view.findViewById(R.id.receiverIp)
+
+        prefs = activity?.getSharedPreferences("settings", android.content.Context.MODE_PRIVATE)!!
+
+        audioSources = arrayOf(
+            getString(R.string.sender_audio_source_current_apps),
+            getString(R.string.sender_audio_source_microphone)
+        )
+
+        usePlaybackCapture?.text = audioSources[selectedAudioSourceIndex]
```
Comment (ortex) [inline app/src/main/java/org/rocstreaming/rocdroid/fragment/SenderFragment.kt:65]: `usePlaybackCapture` defined as non-null, so you don't need null-safe operator `?` here

**Comment by ortex** [comment]
LGTM!

**Review by gavv — APPROVED** [review]

**Pull request merged**